specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -749,6 +749,44 @@
         "x-ms-odata": "#/definitions/DeploymentExtendedFilter"
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/{resourceProviderNamespace}/register": {
+      "post": {
+        "tags": [
+          "Providers"
+        ],
+        "operationId": "Providers_Register",
```

Comment:
The operation ID is going to be the method name in the SDK, so we should have a unique name for it. The current one is already used by an existing API.